protocol/package-lock.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
this is a side effect of the makefile doing an `npm i`. i changed it to use yarn and deleted this file